### PR TITLE
Reformat using alejandra

### PR DESCRIPTION
**Pull request by drupol**
This PR:

- [x] Add the `formatter` output attribute
- [x] Reformat nix files using `nix fmt` command

### CODE REVIEW

**Comment by figsoda** [comment]
I personally prefer nixpkgs-fmt, but I will leave the decision to @Mic92 

also it is probably best to set the formatter in flake.nix if we want to use a specific formatter

File: flake.nix [hunk 1]
```diff
@@ -4,11 +4,20 @@
...
-  outputs = { self, nixpkgs, flake-utils }:
+  outputs = {
+    self,
+    nixpkgs,
+    flake-utils,
+  }:
     flake-utils.lib.eachDefaultSystem (system: {
-      packages.nix-update = nixpkgs.legacyPackages.${system}.callPackage self {
-        src = self;
+      formatter = nixpkgs.legacyPackages.${system}.alejandra;
```
Comment (drupol) [inline flake.nix:13]: I added the formatter here.

**Comment by drupol** [comment]
> I personally prefer nixpkgs-fmt, but I will leave the decision to @Mic92

It would be so nice to have only one formatter...

File: flake.nix (hunk 1 above)
Comment (figsoda) [reply flake.nix:13]: My bad, I didn't see that at the first glance. edited the comment

**Comment by SuperSandro2000** [comment]
I don't like alejandra either because it is doing almost all things differently on purpose.

File: tests/testpkgs/savanna.nix [hunk 2]
```diff
@@ -1,5 +1,9 @@
-{ lib, stdenv, fetchurl, gamin }:
-
+{
+  lib,
+  stdenv,
+  fetchurl,
+  gamin,
+}:
```
Comment (SuperSandro2000) [inline tests/testpkgs/savanna.nix:6]: For example here:
- Why does it enforce multiple lines? Also the commas in the end of the line looks weird.
- Where did the extra new line go?

File: tests/testpkgs/savanna.nix [hunk 3]
```diff
@@ -9,7 +13,7 @@ stdenv.mkDerivation rec {
     sha256 = "0ajc9h023vzpnlqqjli4wbvs0q36nr5p9msc3wzbic8rk687qcxc";
   };
 
-  buildInputs = [ gamin ];
+  buildInputs = [gamin];
```
Comment (SuperSandro2000) [inline tests/testpkgs/savanna.nix:16]: Why drop the spaces here? It makes lists more cramped and everything feels much tighter which makes the code harder to read.

File: shell.nix [hunk 4]
```diff
@@ -1,9 +1,7 @@
-{ pkgs ? import <nixpkgs> { } }:
-
+{pkgs ? import <nixpkgs> {}}:
```
Comment (SuperSandro2000) [inline shell.nix:1]: same here, spaces improve readability but alejandra just randomly decided to remove the first and last spaces while keeping the ones around the question mark.

**Comment by drupol** [comment]
Closing in favor of #127

**Pull request closed without merging**